sys/include/bcd.h

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2017 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    sys_bcd Binary coded decimal
+ * @ingroup     sys
+ * @brief       Library to de- and encode binary coded decimals
+ * @{
+ *
+ * @file
+ * @brief       BCD definitions
+ *
+ * @author  Martine Lenders <m.lenders@fu-berlin.de>
+ */
+#ifndef BCD_H
+#define BCD_H
+
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Converts a byte to a binary coded decimal
+ *
+ * @param[in] byte  A byte
+ *
+ * @return  A binary coded decimal (4 MSB = 10s, 4 LSB = 1s)
+ */
+static inline uint8_t bcd_from_byte(uint8_t byte)
+{
+    return byte + (6 * (byte / 10));
```

Comment:
Though this is (more) elegant, it's not as intuitive as `((byte / 10) << 4) | (byte % 10)`, which makes the conversion IMHO quite explicit. I don't suggest to change this, but it might be worth a comment (or example)?